yt/yt/server/queue_agent/queue_static_table_exporter.cpp

```diff
@@ -446,13 +446,16 @@ class TQueueExportTask
         chunkSpecFetcher->Add(
             QueueObject_.ObjectId,
             QueueObject_.ExternalCellTag,
-            QueueObject_.ChunkCount);
+            // XXX(achulkov2, gritukan): YT-11825
+            /*chunkCount*/ -1);
 
         WaitFor(chunkSpecFetcher->Fetch())
             .ThrowOnError();
 
         ChunkSpecs_ = chunkSpecFetcher->GetChunkSpecsOrderedNaturally();
         YT_LOG_DEBUG("Finished fetching chunk specs (Count: %v)", ChunkSpecs_.size());
+
+        YT_LOG_FATAL_UNLESS(std::ssize(ChunkSpecs_) == QueueObject_.ChunkCount, "Fetched chunk spec count mismatch: expected %v, received %v", QueueObject_.ChunkCount, ChunkSpecs_.size());
```

Comment:
Are you sure that dynamic stores will not make these values different?

Reply:
Sadly, you are right in case of @enable_dynamic_store_read = %true. I don't see a way to make this check work, so I removed it.